tests/components/recorder/test_websocket_api.py

```diff
@@ -3173,3 +3173,27 @@ async def test_adjust_sum_statistics_errors(
         stats = statistics_during_period(hass, zero, period="hour")
         assert stats != previous_stats
         previous_stats = stats
+
+
+async def test_recorder_recorded_entities(
+    recorder_mock: Recorder, hass: HomeAssistant, hass_ws_client: WebSocketGenerator
+) -> None:
```

Comment:
The point of the new command is that the result should exclude entities which are not recorded. I think that should be covered by the test, maybe set a state which is filtered out by recorder settings?

Reply:
Good idea.  added in 81a58cdfddc3ef0b3da00582ec299871eb88ad28